toppingmaker/exportsettings.py

```diff
@@ -69,28 +80,30 @@ def set_setting_values(
         name: str = None,
         export=True,
         categories=None,
+        style: str = None,
     ) -> bool:
         """
         Appends the values (export, categories) to an existing setting
```

Comment:
Adjust docstring (remove "export, categories" or add style)

Reply:
But it does not append the `style` to a setting. A setting is identified by the `style`.
For better readability it would make sense to have style before. Maybe I can change that (since I guess only Model Baker uses this libary at the moment)

Reply:
Maybe in the end of those implementations I will "strähl" it nomal.

Reply:
Can we rename `style` to `style_name` or `style_id` all over the place for better readability?